Telerik.JustMock.DemoLib/Objects/ClassWithDynamicDataMethod.cs

```diff
@@ -0,0 +1,35 @@
+/*
+ JustMock Lite
+ Copyright © 2010-2018 Telerik EAD
```

Comment:
This is a new class. The corresponding copyright is Copyright © 2018 Telerik EAD

Reply:
@mihail-vladov fixed with [f1371a1](https://github.com/telerik/JustMockLite/pull/41/commits/f1371a136290300edd1b6399e76d23fd0640ce5c)